users/tests.py

```diff
@@ -1,3 +1,65 @@
-from django.test import TestCase
+from unittest.mock import patch, MagicMock
 
-# Create your tests here.
+import jwt, json, requests
+
+from django.test   import TestCase, Client
+from django.conf   import settings
+
+from .models import User
+
+class UserTest(TestCase):
+    def setUp(self):
+        User.objects.create(
+            id            = 1000,
+            kakao_id      = 123456789,
+            email         = 'test@kakao.com',
+            nickname      = 'nick',
+            thumbnail_url = 'http://tinypic.com/test.jpg',
+        )
+
+    def tearDown(self):
+        User.objects.all().delete()
+
+    @patch("users.views.requests")
+    def test_success_kakaologinview(self, mocked_requests):
+        client = Client()
+
+        class MockedResponse:
+            def json(self):
+                return {
+                    'id': 1000,
+                    'kakao_account': {
+                        'email' : 'test2@kakao.com',
+                        'profile': {
+                            'nickname'           : 'nick',
+                            'thumbnail_image_url': 'http://tinypic.com/test.jpg'
+                            }
+                        }
+                    }
+
+        mocked_requests.get = MagicMock(return_value = MockedResponse())
+        headers             = {"HTTP_Authorization": "123456789"}
+        response            = client.get("/users/login/kakao", **headers)
+        access_token        = jwt.encode({'id': User.objects.latest('id').id}, settings.SECRET_KEY, settings.ALGORITHM)
+
+        self.assertEqual(response.json(), {'cream_token': access_token})
+        self.assertEqual(response.status_code, 201)
```

Comment:
```suggestion
        self.assertEqual(response.json(), {'cream_token': access_token})
```

좋은 테스트 코드가 있다면 추후에 어떠한 경우로 인해 변경된 코드를 검증함으로써 이로 인해 생길 수 있는 문제를 해결할 수 있습니다. 또한 실제 코드가 변경되면 테스트 코드 역시 변경이 필요할 수 있는데, 이러한 이유로 우리는 테스트 코드 역시, 늘 강조하는 바와 같이 가독성있게 작성할 필요가 있습니다!

그렇기에 테스트를 작성하는 경우에는 다음을 준수하는 것이 좋습니다.

- 1개의 테스트 함수에 대해 assert를 최소화하라
- 1개의 테스트 함수는 1가지 개념 만을 테스트하라

위에 따라 45, 46 line 중 하나만 가지고도 충분히 검증할 수 있을 것 같습니다.

Reply:
리뷰해주신대로 수정했습니다